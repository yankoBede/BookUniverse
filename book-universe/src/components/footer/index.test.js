import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './index'

describe('Footer Component', () => {
    it('should render react course link and softuni link', () => {
        const tree = renderer.create(
                <Footer/>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})