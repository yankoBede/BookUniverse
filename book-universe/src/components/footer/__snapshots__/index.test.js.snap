// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render react course link and softuni link 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <p
      className="text-muted text-center"
    >
      <a
        href="https://softuni.bg/trainings/3023/reactjs--june-2020/internal"
      >
        React Course
      </a>
      @
      <a
        href="https://www.softuni.bg"
      >
        SoftUni
      </a>
    </p>
    <p
      className="text-muted text-center"
    >
      All rights reserved Â© 2020
    </p>
  </div>
</footer>
`;
